README.md

```diff
@@ -9,3 +12,10 @@ TODO: add link here
 
 ## Reference Implementations
 TODO: add link here
+
+## Why are we doing this?
+* If this becomes popular, frameworks and other services will automatically pass trace IDs through for correlated requests. This would prevent traces from hitting dead ends when a request reaches an un-instrumented service.
+* Once aligned on a header name, we can ask for a CORS exception from the W3C. This would allow browsers to attach trace IDs to requests and submit tracing data to a distributed tracing service.
+* Loggers can reliably parse trace / span IDs and include them in logs for correlation purposes.
+* Customers can use multiple tracing solutions (Zipkin + New Relic) at the same time and not have to worry about propagating two sets of context headers.
```

Comment:
The way I interpret this point is: trace-id is passed unchanged through apps using Zipkin and NewRelic.
  * corollary: both Zipkin and NewRelic must share the same trace-id
  * corollary: only tracing systems using the exact same format of trace-id can be compatible under this spec

Reply:
merging this as it represents historical context of the spec. We will probably need to update this at some point when headers are finished definition, ex if we are able to meet this goal or if our header becomes secondary (backup header). We can't know yet if we can reach this goal basically, and for which vendors, but it is good to have a goal